src/ActionModal/components/SendModal.vue

```diff
@@ -286,7 +286,8 @@ export default {
   },
   methods: {
     open(denom = undefined) {
-      this.selectedToken = denom || this.selectedToken
+      this.selectedToken =
+        denom || this.balances.find(({ denom }) => denom).denom
```

Comment:
what should this do? `this.balances.find(({ denom }) => denom).denom` this part doesn't do anything

Reply:
yes, it returns the first denom from balances, which makes total sense, because if the user hasn't selected any token yet, then he is going with the first balance (didn't touch the dropdown)

Reply:
ah I get it